core/DataAccess/LogAggregator.php

```diff
@@ -377,21 +377,14 @@ public function generateQuery($select, $from, $where, $groupBy, $orderBy, $limit
             $logTablesProvider = $this->getLogTableProvider();
             $logTablesProvider->setTempTable(new LogTableTemporary($segmentTable));
 
-            foreach ($logTablesProvider->getAllLogTables() as $logTable) {
-                if ($logTable->getDateTimeColumn()) {
-                    $whereTest = $this->getWhereStatement($logTable->getName(), $logTable->getDateTimeColumn());
-                    if (strpos($where, $whereTest) === 0) {
-                        // we don't need to apply the where statement again as it would have been applied already
-                        // in the temporary table... instead it should join the tables through the idvisit index
-                        $where = ltrim(str_replace($whereTest, '', $where));
-                        if (stripos($where, 'and ') === 0) {
-                            $where = substr($where, strlen('and '));
-                        }
-                        $bind = array();
-                        break;
-                    }
+            if (strpos($where, $segmentWhere) === 0) {
```

Comment:
we basically only want to remove `log_visit.visit_last_action_time>= ? AND log_visit.visit_last_action_time<= ? AND log_visit.idsite IN (...)"` from a potential where statement (because it is already applied in original statement) but not any other where clause.